Code/Engine/JsonManager.cpp

```diff
@@ -0,0 +1,74 @@
+#include "sadpch.h"
+
+#include "JsonManager.h"
+
+using json = nlohmann::json;
+//Constructor
+sad::JsonManager::JsonManager(){}
```

Comment:
Does this need a constructor? It looks like most of what we're using is static from this class.